<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border-image:url(:/qrc/LoginBack.jpg);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>426</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnZoom">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border-image: url(:/qrc/button_min);
	background-color: rgba(255, 170, 225, 120);
}

QPushButton:hover
{
	border-image: url(:/qrc/button_min_hover);
}

</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border-image: url(:/qrc/button_close);
	background-color: rgba(255, 170, 225, 120);
}

QPushButton:hover
{
	border-image: url(:/qrc/button_close_hover);
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="labTitle">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>321</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>教室管理系统</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="frame_7">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>341</width>
           <height>81</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="editUsername">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,70%);
border:1px solid #b9babb;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnRegist">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>10</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,70%);
border:1px solid #b9babb;</string>
          </property>
          <property name="text">
           <string>注册账号</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editPassword">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,70%);
border:1px solid #b9babb;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnFindPassword">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>50</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,70%);
border:1px solid #b9babb;</string>
          </property>
          <property name="text">
           <string>修改密码</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="labHead">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>71</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>:/qrc/head.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QCheckBox" name="btnRemember">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>0</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 170, 255)</string>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="btnAuto">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>0</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 170, 255)</string>
         </property>
         <property name="text">
          <string>自动登录</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnSafeLogin">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>30</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    background-color:rgba(255, 255 , 255,70%);
}

QPushButton:hover
{
    color:rgb(97 , 179 , 246);
}

QPushButton:pressed
{
    color:rgb(0 , 109 , 176);
}
</string>
         </property>
         <property name="text">
          <string>安全登录</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
